In diesem Projekt will ich die Angebotenen Rentenversicherungen miteinander vergleichen. Die PDF mit den Angeboten liegt in "C:\Users\Benja\OneDrive\Desktop\Ordnerstruktur\Finanzen\Privat\Versicherung\neue Versicherungen\Andy\230604".

Was will ich genau machen:
1) Ausrechnen, wie viel Geld ich theoretisch hätte, wenn ich das eingezahlte Geld selbst mit den angegebenen Renditewerten angelegt hätte. Den Betrag dann mit den Kapital CRK Werten vergleichen.
2) Den Vergleich machen, wenn ich monatlich etwas von diesem Geld nehme, wie sich dann das Geld im Vergleich zu den Versicherungen ändert.
3) Die einzelnen Versicherungen recherchieren und die einzelnen Werte in der PDF recherchieren.
4) was ist meine Rentenlücke. (mit Inflation)

# 1) Was ich theoretisch erspart hätte.

- Annahme: Rendite ist einmal pro Jahr

In [7]:
def kapital(einzahlung: int, rendite: int, jahre: int) -> int:
    """einzahlung = monatliche einzahlung, rendite = jährliche rendite in prozent, jahre = jahre eingespart""" 
    def temp(kapital_zuvor: int, einzahlung: int, rendite: int) -> int:
        return (kapital_zuvor + 12*einzahlung) * (1+rendite/100)
    
    kapital=0
    for jahr in range(jahre):
        kapital = temp(kapital_zuvor=kapital, einzahlung=einzahlung, rendite=rendite)
    return kapital

In [11]:
kapital(150, 2, 41)

114952.00194224031

Für eine Rendite von 2% bei 41 Jahren Einsparen:
 - Volkswohlbund: 91683.00 €
 - ERGO: 93,349.00 €
 - Die Continenthale: 93,455.07 €
 - HDI: 90,989.00 €
 - SwissLife: 85,612.18 €
 - ohne: 114,952.00 €

Das sind jeweils wie viel Prozent von selbst anlegen:
 - Volkswohlbund: 79.76 %
 - ERGO: 81.2 %
 - Die Continenthale: 81.3 %
 - HDI: 79.15 %
 - SwissLife: 74.48 %

In [12]:
kapital(150, 6.43, 41)

353694.54212246614

Für eine Rendite von CRK bei 41 Jahren Einsparen:
 - Volkswohlbund: 302,041.00 €
 - ERGO: 308,442.00 €
 - Die Continenthale: 306,927.00 €
 - HDI: 304,949.00 €
 - SwissLife: 282,045.84 €
 - ohne: Zum Vergleich wurde die Durchschnittsrendite des MSCII World genommen, 6.43% (Quelle; "https://vivetmaximum.de/etf-rendite/"): 353,694.54 €
https://vivetmaximum.de/etf-rendite/

Das sind jeweils wie viel Prozent von selbst anlegen:
 - Volkswohlbund: 85.4 %
 - ERGO: 87.2 %
 - Die Continenthale: 86.78 %
 - HDI: 86.22 %
 - SwissLife: 79.74 %

3708.7200000000003